Duke.java

```diff
@@ -0,0 +1,185 @@
+import java.util.Scanner;
+
+public class Duke {
+    // static variables
```

Comment:
Avoid repeating the obvious, consider removing the comment as the code is self-explanatory.